velox/dwio/parquet/reader/ParquetBloomFilter.h

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include "velox/dwio/parquet/reader/BloomFilter.h"
+#include "velox/type/AbstractBloomFilter.h"
+
+namespace facebook::velox::parquet {
+
+class ParquetBloomFilter final : public AbstractBloomFilter {
```

Comment:
Any other Velox XXXBloomFilter that will extend AbstractBloomFilter? It's good to have the AbstractBloomFilter interface, but is it necessary to have a separate ParquetBloomFilter? The original Arrow BloomFilter class can have other sub classes, but I don't see if other Velox XXXBloomFilter would have different implementation than ParquetBloomFilter.

Reply:
It's more for dependency management.  We cannot have parquet dependency in `velox/type`

Reply:
From previous discussion on https://github.com/facebookincubator/velox/issues/4414, we expect to use the same interface for ORC/DWRF bloom filter implementation.